### ISSUE TITLE
NEW, OPEN menuitem issue

### ISSUE THREAD

**Issue by DinyaZ** [twinbasic/twinbasic#788]
**To Reproduce**
1. Open the Helloworld (or any other) demoapp.
2. It works, fine.
3. Then I push the FILE / NEW or the OPEN menuitem
4. I push there the CANCEL
5. The previously opened app already disappeared, I get back an empty page. 

**Expected behavior**
Dont close the already opened app directly, only after I began to open another one or start a new. Make it possible to go back with cancel.

**Desktop (please complete the following information):**
 - OS: Windows7 64bit
 - twinBASIC compiler version beta10

**Comment by EduardoVB** [comment]
Not sure if this was with the VSCode IDE or the new one, but most probably no longer valid. Anyone can confirm?

**Comment by Kr00l** [comment]
Most probably VS code. Can't repro on new IDE.
If there remains an issue somewhere a new issue may be created.

**Comment by fafalone** [comment]
This behavior is still the same; it's worth discussing whether it should be.

**Comment by fafalone** [comment]
I'll make a new one.
